src/net/rpc.rs

```diff
@@ -1260,6 +1260,7 @@ impl ConversationHttp {
                     let key = ClarityDatabase::make_key_for_account_balance(&account);
                     let burn_block_height = clarity_db.get_current_burnchain_block_height() as u64;
                     let v1_unlock_height = clarity_db.get_v1_unlock_height();
+                    let v2_unlock_height = clarity_db.get_v2_unlock_height();
```

Comment:
Could we place also add `pox_v2_unlock_height` as a property to the `/new_block` event payload, similar to the current `pox_v1_unlock_height` property? I think it's probably a one-line change, and will help the API report correct locked balance amounts.